src/bindings/python/docs/test_examples.md

```diff
@@ -35,6 +45,31 @@ To run full test suite one can utilize `tox` command:
 tox
 ```
 
+### Check codestyle of Python API
+There are two packages used in the project to check codestyle of python code: *mypy* and *flake8*.
+Additionally OpenVINO™ uses the custom configuration file to exclude some strict rules.
+
+For checking the codestyle of Python API 2.0 execute this command:
+```
+python -m flake8 ./src/openvino/ --config=setup.cfg
+python -m mypy ./src/openvino --config-file ./setup.cfg
+```
+For checking the codestyle of compatibility nGraph Python API execute this command:
+```
+python -m flake8 ./src/compatibility/ngraph/ --config=setup.cfg
+python -m mypy ./src/compatibility/ngraph --config-file ./setup.cfg
+```
+For checking the codestyle of compatibility InferenceEngine Python API execute this command:
```

Comment:
```suggestion
To check the codestyle of InferenceEngine Python API compatibility, run the following commands:
```

Reply:
done

Reply:
For more clarity I would suggest:
To check the codestyle of the "compatibility InferenceEngine Python API", run the following commands: